setup.py

```diff
@@ -16,6 +16,7 @@ def get_requirements():
         "flake8==3.9.2",
         "isort==5.9.2",
         "pytest>=6.2.4",
+        "importlib-metadata>=1.1.0,<4.3;python_version<'3.8'",
```

Comment:
Is `python_version` for the supported python version or has a different meaning? I am asking because we have been supporting till 3.7, 3.8 and 3.9

Reply:
This is a good point, I thought it was supported and I got this directly from [flake8](https://github.com/PyCQA/flake8/blob/b220ee957aca37ae5919509f6f25dd571a836d52/setup.cfg#L44), but they included this on `setup.cfg` rather than on `requirements.txt` or `setup.py`.

When I searched for that package on pypi, I actually founded a package [python_version](https://pypi.org/project/python_version/), so including this on `setup.py` instead of `setup.cfg` might install that redundant (?) package I think. I will refactor this.